--- /usr/local/lib/python2.7/dist-packages/swift/common/utils.py 2014-09-06 15:45:06.505798157 +0200
+++ utils.py    2016-01-05 18:41:55.282627124 +0100
***************
*** 572,578 ****
          # fallocate is preferred because we need the on-disk size to match
          # the allocated size. Older versions of sqlite require that the
          # two sizes match. However, fallocate is Linux only.
!         for func in ('fallocate', 'posix_fallocate'):
              self.func_name = func
              self.fallocate = load_libc_function(func, log_error=False)
              if self.fallocate is not noop_libc_function:
--- 572,578 ----
          # fallocate is preferred because we need the on-disk size to match
          # the allocated size. Older versions of sqlite require that the
          # two sizes match. However, fallocate is Linux only.
!         for func in ('posix_fallocate', 'fallocate'):
              self.func_name = func
              self.fallocate = load_libc_function(func, log_error=False)
              if self.fallocate is not noop_libc_function:
 
